src/functions/authoriser/application/getExaminerRole.ts

```diff
@@ -16,13 +16,13 @@ export default async function getEmployeeRole(employeeId: EmployeeId): Promise<s
     Key: {
       staffNumber: employeeId,
     },
-    ProjectionExpression: role,
+    ProjectionExpression: examinerRole,
```

Comment:
Isn't it will named `role` in DDB? Would have thought renaming here would break things?

Reply:
Yes actually, thanks.